bun/lib/dependabot/bun/update_checker/version_resolver.rb

```diff
@@ -419,18 +535,22 @@ def satisfying_versions
               rescue Gem::Requirement::BadRequirementError
                 false
               end
+            end.map do |versions_with_details| # rubocop:disable Style/MultilineBlockChain
+              # Return just the version
+              version, = versions_with_details
+              version
             end
```

Comment:
nit, you could try something like this:

```suggestion
            end.map(&:first) # Return only the version
```

Reply:
Created a PR for that separately: 

- https://github.com/dependabot/dependabot-core/pull/11999